$NetBSD: patch-CMakeLists.txt,v 1.3 2023/05/07 10:01:26 wiz Exp $

PythonInterp is deprecated and does not work correctly.
Do not set -O3 optimization flags.

--- CMakeLists.txt.orig	2023-04-28 06:55:41.000000000 +0000
+++ CMakeLists.txt
@@ -41,7 +41,7 @@ message(STATUS "Using ClangConfig.cmake 
 
 # Locate Python and check its version.
 #
-find_package(PythonInterp 3.6 REQUIRED)
+find_package(Python3 COMPONENTS Interpreter Development)
 
 # Locate pytest
 execute_process(COMMAND ${PYTHON_EXECUTABLE} -m pytest --version
@@ -170,8 +170,6 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU"
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-error=maybe-uninitialized")
   endif()
 
-  set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O3")
-  set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELEASE} -O3 -g")
 endif()
 
 ###############################################################################
